bindings/nodejs/examples/how_tos/accounts_and_addresses/check_balance.ts

```diff
@@ -18,12 +18,11 @@ async function run() {
         const account = await wallet.getAccount('Alice');
 
         // Sync new outputs from the node.
-        const balance = await account.sync();
-        console.log('Account balance:', balance);
+        const _balance = await account.sync();
```

Comment:
Assigned but never used

Reply:
Was just trying to show that sync also returns the balance. I also updated the docs to explain that better. At least I tried 😆

Reply:
Can I somehow let the linter ignore that?
Or any other ideas?

Reply:
You can add `// prettier-ignore` above

Reply:
```suggestion
        // eslint-disable-next-line no-unused-vars
        const _balance = await account.sync();
```

Like this?

Reply:
I think so, but needs to be tested, also would remove the underscore from _balance then

Reply:
Let's try :trollface: 

```suggestion
        // eslint-disable-next-line no-unused-vars
        const syncBalance = await account.sync();
```